### ISSUE TITLE
`request_anim` invalidates the entire window

### ISSUE THREAD

**Issue by jneem** [xi-editor/druid#867]
Once #817 is merged, we'll have support for partial invalidation. However, `request_anim` doesn't make use of it yet. This can be seen in `druid/examples/invalidation.rs`, where the whole window gets repeatedly invalidated while the scrollbar is fading out.

**Comment by raphlinus** [comment]
Yes, this is something that worried me. Agree it should be fixed, and `request_anim` shouldn't do any invalidation itself, as it doesn't know what to invalidate.
